lib/router.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_example_app/details_screen.dart';
+import 'package:flutter_example_app/tracks_data.dart';
+import 'package:flutter_example_app/tracks_screen.dart';
+
+class Routes {
+  static const tracksScreen = '/';
+  static const detailsScreen = '/details';
+}
+
+class Router {
```

Comment:
please rename this class, because there will be a conflict with package:flutter/src/widgets/router.dart, use, for example, AppRouter or something like this